x-pack/platform/test/alerting_api_integration/spaces_only/tests/alerting/group2/update.ts

```diff
@@ -573,7 +570,7 @@ export default function createUpdateTests({ getService }: FtrProviderContext) {
           .send(getTestRuleData())
           .expect(200);
 
-        const longInvestigationGuideBlob = 'a'.repeat(MAX_ARTIFACTS_INVESTIGATION_GUIDE_LENGTH + 1);
+        const longInvestigationGuideBlob = 'a';
```

Comment:
I guess this test is not needed anymore.

Have we investigated what was causing the issue?

I'm also curious to know why we had a length limit for the investigation guide and why we decided to remove it.

Reply:
@cesco-f Here's the [thread](https://elastic.slack.com/archives/C03VC331C9Y/p1760704944146049) for more context.

There were discussions about either removing the limit or finding a good limit to use. The default max ES limit is [100MB](https://www.elastic.co/docs/deploy-manage/production-guidance/general-recommendations#maximum-document-size) according to the documentation. 1MB was the suggestion at the end of the discussion, but in general the direction was to completely remove the limit. Even if we set a 1MB limit, we would need to convert it to number of characters. This depends on the encoding for the different languages. For English it would be 1 million characters, for Greek 0.5 million characters. So I decided to simply remove the limit.

Reply:
> Have we investigated what was causing the issue?

The issue described on the SDH about UI being locked and not being able to delete the text when reaching the 1000 character limit is not reproducible on main. Client confirmed that it is working on 9.1.5 fine. So I didn't investigate what was causing the issue and what fixed it. 

So the PR is about removing the character limit. I'll backport to proper versions as well

Reply:
I have a question regarding backporting btw. Shall I be selecting `9.1.0` or `9.1.7`?

Reply:
>I have a question regarding backporting btw. Shall I be selecting 9.1.0 or 9.1.7?

If you want to backport to the next 9.1.x version you should select 9.1.7.

Reply:
> Not sure if it will ever happen but what error the user will see if they hit that limit?

I haven't tried it. Let me check where the limit for the kibana router is defined so that I can test it and see what the user will see.

Reply:
> investigate why it's working in 9.1.5 and backport that change to the affected version

That's what I would do if I hadn't decided to remove the limit. Let me check out what the user will see it they hit the Kibana react router. If nothing looks broken, I would keep the PR as is.

Reply:
@cesco-f I tested it by adding `server.maxPayload: 5120` in kibana.dev.yml, added a longer text and that's how it looks like. No broken experience for the user.

<img width="2466" height="1464" alt="Screenshot 2025-10-24 at 12 52 10" src="https://github.com/user-attachments/assets/93d7d7c1-9576-4a81-97a7-99c75f07fe1c" />

Reply:
I don't know how likely is that a user gets to see that message, but it's not very user friendly.

I still think a solution like this is better: https://github.com/elastic/kibana/pull/240472.

I fixed the infinite loop that was causing the SDH issue, and we can set the limit to whatever we think is reasonable.